docs/DeveloperGuide.md

```diff
@@ -720,101 +765,88 @@ testers are expected to do more *exploratory* testing.
                   dr/13/09/22 - 15/09/22 ng/1 rq/Apply for room service `<br>
-      Expected: Guest add successfully
+      Expected: Guest add successfully.
 
    3. Test case: `add n/John Doe p/98765431 e/johnd@nus.com rm/06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
       Expected: No guest is added,
-      because the name is already in GuestBook. Error details shown in the status message.
+      because the **name is already in GuestBook**. Error details shown in the status message.
          Status bar remains the same.
 
    4. Test case: `add n/Peter p/98765431 e/johnd@nus.com rm/05-73
          dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
          Expected: No guest is added,
-         because the room is already in GuestBook. Error details shown in the status message.
+         because the **room is already in GuestBook**. Error details shown in the status message.
          Status bar remains the same.
 
    5. Test case: `add n/John@y Doe p/98765431 e/johnd@nus.com rm/06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
-      Expected: No guest is added, because the name is invalid. Error details shown in the status message.
+      Expected: No guest is added, because the **name** is invalid. Error details shown in the status message.
       Status bar remains the same.
 
    6. Test case: `add n/Johnny Doe p/+65431 e/johnd@nus.com rm/06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
-      Expected: No guest is added, because the phone number is invalid. Error details shown in the status message.
+      Expected: No guest is added, because the **phone number** is invalid. Error details shown in the status message.
       Status bar remains the same.
 
    7. Test case: `add n/Johnny Doe p/98765431 e/nus.com rm/06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
-      Expected: No guest is added, because the email address is invalid. Error details shown in the status message.
+      Expected: No guest is added, because the **email address** is invalid. Error details shown in the status message.
       Status bar remains the same.
 
    8. Test case: `add n/Johnny Doe p/98765431 e/johnd@nus.com rm/!06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
-      Expected: No guest is added, because the room is invalid. Error details shown in the status message.
+      Expected: No guest is added, because the **room** is invalid. Error details shown in the status message.
       Status bar remains the same.
 
    9. Test case: `add n/Johnny Doe p/98765431 e/johnd@nus.com rm/06-73
                   dr/13/09/22 - 13/09/23 ng/1 rq/Kill the insect `<br>
-      Expected: No guest is added, because the date range is invalid. Error details shown in the status message.
+      Expected: No guest is added, because the **date range** is invalid. Error details shown in the status message.
```

Comment:
I dont think this dateRange is invalid? maybe replace with 13/09/22 - 10/09/22 or something

Reply:
Yes fixed.